splunk-cloudwatch-logs-processor/index.js

```diff
@@ -112,6 +113,11 @@ const getSplunkLogger = (parsed, context, callback) => {
 
             console.log('Data retrieved from SSM:', JSON.stringify(ssmData, null, 2));
 
+            // Make splunk_json optional
```

Comment:
Please update comment for folks who are not JS coders:

```suggestion
            // Remove optional splunk_json from InvalidParameters if it exists
```